tools/doxygen/doxygen.cmake

```diff
@@ -419,7 +419,19 @@ function(make_doxygen_target modules_var)
     set_target_properties("DOXY-Inviwo" PROPERTIES FOLDER "doc" EXCLUDE_FROM_ALL TRUE)
     add_dependencies("DOXY-ALL" "DOXY-Inviwo")
 
-    # Help, used for the help inside invowo
+
+    add_custom_target("DOXY-generate-processor-previews"
+        COMMAND inviwo --save-previews "${output_dir}/inviwo/html" --quit
+        WORKING_DIRECTORY ${output_dir}
+        COMMENT "Generating Inviwo API documentation with Doxygen"
```

Comment:
Comment should say
Generate preview images of processors used in Inviwo Doxygen API documentation.